schema/crdb/bp-omicron-zone-filesystem-pool-not-null/up.sql

```diff
@@ -0,0 +1,7 @@
+-- This migration will fail if there are any remaining bp_omicron_zone entries
+-- with a NULL filesystem_pool. We've confirmed via the `omdb reconfigurator
+-- archive` outputs from R13 upgrades that all deployed systems have
+-- fully-populated `filesystem_pool` values.
+ALTER TABLE omicron.public.bp_omicron_zone
+    ALTER COLUMN filesystem_pool
+    SET NOT NULL;
```

Comment:
This migration will fail if there are any blueprints left with any zones with a `NULL` filesystem pool. This is why the R13 upgrade instructions included a step to use `omdb reconfigurator archive`, which removes all but the target blueprint.